packages/devtools_app/lib/src/config_specific/flutter/import_export/import_export.dart

```diff
@@ -75,5 +63,29 @@ abstract class ExportController {
 
   const ExportController.impl();
 
-  void downloadFile(String filename, String contents);
+  String get filename {
```

Comment:
this shouldn't be a getter because it returns a new value every time it is called. Getters should whenever possible behave like fields which have the same value if you call them twice in a row.
something like
```dart
String generateFilename()
```
would make the intent more clear.

Reply:
done